src/test/java/com/zepben/evolve/services/network/testdata/TestDataCreators.kt

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright 2020 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package com.zepben.evolve.services.network.testdata
+
+import com.zepben.evolve.cim.iec61968.assetinfo.OverheadWireInfo
+import com.zepben.evolve.cim.iec61968.assetinfo.WireInfo
+import com.zepben.evolve.cim.iec61968.assets.AssetOwner
+import com.zepben.evolve.cim.iec61968.common.Organisation
+import com.zepben.evolve.cim.iec61968.metering.Meter
+import com.zepben.evolve.cim.iec61968.metering.UsagePoint
+import com.zepben.evolve.cim.iec61968.operations.OperationalRestriction
+import com.zepben.evolve.cim.iec61970.base.core.*
+import com.zepben.evolve.cim.iec61970.base.wires.*
+import com.zepben.evolve.services.customer.CustomerService
+import com.zepben.evolve.services.network.NetworkService
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
+import java.util.function.Consumer
+
+/**
+ * Class that generates some networks for testing with.
+ */
+fun createSourceForConnecting(network: NetworkService, id: String, numTerminals: Int, phaseCode: PhaseCode): EnergySource =
```

Comment:
Noise around `TestDataCreators` imports can be removed by wrapping it in an object like `DifferenceNetworks`

Reply:
yeah the reason i preferred this was because once you wrap it in an object you can no longer do:
```java
import com.zepben.evolve.services.network.testdata.TestDataCreators.*
```
in a kotlin file when you need the static methods (e.g createAcLineSegment, createSource, createNode and others need to be static)

whereas if you have no object java things can still access it per normal by the TestDataCreatorsKt class, and kotlin can access it directly under testdata.

Reply:
i can fix some of the noise however, as the imports are no longer necessary for FillFields and SingleLineSinglePhase as they are in the same package.

Reply:
Yep, stupid Kotlin limitation ☹️